### PR TITLE
docs: Release notes for 0.13.9

### PR DESCRIPTION
**Pull request by determined-dsw**
## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

File: docs/release-notes.txt [hunk 1]
```diff
@@ -8,6 +8,24 @@
...
+-  Commands: Support configuring shmSize for commands (e.g., notebooks,
+   shells, TensorBoards) in :ref:`command configurations
+   <command-notebook-configuration>`.
+
+**Bug Fixes**
+
+-  API: Fix a bug that caused the webUI's log viewer to fail to render
+   older pages of trial logs.
+-  WebUI: Fix a bug in opening TensorBoards from the experiment list
+   page via multiple batch operations.
```
Comment (vishnu2kmohan) [inline docs/release-notes.txt:27]:
```suggestion
   page via batch selection.
```

File: docs/release-notes.txt [hunk 2]
```diff
@@ -8,6 +8,24 @@
...
+**Improvements**
+
+-  Commands: Support configuring shmSize for commands (e.g., notebooks,
+   shells, TensorBoards) in :ref:`command configurations
+   <command-notebook-configuration>`.
+
+**Bug Fixes**
+
+-  API: Fix a bug that caused the webUI's log viewer to fail to render
+   older pages of trial logs.
```
Comment (vishnu2kmohan) [inline docs/release-notes.txt:25]:
```suggestion
   previous pages of trial logs.
```

**Review by vishnu2kmohan — APPROVED** [review]
Made a couple of suggestions but otherwise LGTM.

**Review by justin-determined-ai — APPROVED** [review]
I made two very small changes. LGTM!

**Pull request merged**